riotfile.py

```diff
@@ -1200,6 +1200,15 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 ),
             ],
         ),
+        Venv(
+            name="aiomysql",
+            pys=select_pys(min_version="3.7", max_version="3.9"),
```

Comment:
```suggestion
            pys=select_pys(min_version="3.7"),
```

Is 3.10 supported?

if we don't define a max, then it'll use the max that we test with (3.10 today)